Constants.js

```diff
@@ -48,10 +48,10 @@ const prettyRole = {
 
 const firebaseErrors = {
   "auth/email-already-in-use": "Email is already in use",
-  "auth/invalid-email": "Invalid email",
+  "auth/invalid-email": "Incorrect username / password",
```

Comment:
Should this be like this since we use email and not username?

```suggestion
  "auth/invalid-email": "Incorrect email / password",
```